application/templates/session-ended.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base/lr_base_layout.html" %}
+{% block content %}
+<main id="content" role="main">
+    {{ super() }}
+    <div id="content">
+        <div class="inner-block">
+            <h1 class="heading-xlarge">Your session has ended</h1>
+            <p class="text">Your session has been ended by the service or has
+            timed out because you haven’t done anything for 20 minutes. We’ve
+            deleted all the details you’ve entered to protect your privacy.</p>
+            <p class="text">You’ll need to <a href="../sign-my-mortgage">start
```

Comment:
Was thinking similar but a bit different, its not a big thing, I was thinking more along the line of flask might have a feature more like the ruby on rails feature where you can:
```
 <td><%= link_to "Show", book %></td>
 <td><%= link_to "Edit", edit_book_path(book) %></td>
```
To generate URLs for you so that you can change the path without affecting anything. I don't really mind.